dali/test/python/operator_1/test_numba_func.py

```diff
@@ -12,19 +12,34 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+import numba
 import numpy as np
 import os
 from nvidia.dali import pipeline_def
 import nvidia.dali as dali
 import nvidia.dali.fn as fn
 import nvidia.dali.types as dali_types
+import platform
+from nose import SkipTest, with_setup
 from test_utils import get_dali_extra_path
 from nvidia.dali.plugin.numba.fn.experimental import numba_function
+from distutils.version import LooseVersion
 
 test_data_root = get_dali_extra_path()
 lmdb_folder = os.path.join(test_data_root, 'db', 'lmdb')
 
 
+def check_env_compatibility():
+    # At present (as of Numba 0.57) there's a bug in LLVM JIT linker that makes the tests fail
+    # randomly on 64-bit ARM platform.
+    #
+    # TODO(michalz): Update the Numba version range when there's a fix - or possibly check
```

Comment:
Please link the relevant issue for the reference.

Reply:
Done.